configs/hand/3d_kpt_sview_rgb_img/internet/interhand3d/res50_interhand3d_all_256x256.py

```diff
@@ -136,7 +136,7 @@
 data_root = 'data/interhand2.6m'
 data = dict(
     samples_per_gpu=16,
-    workers_per_gpu=2,
+    workers_per_gpu=1,
```

Comment:
Is this change necessary?

Reply:
Yes, the data of interhand2.6m v1.0 is too massive. It will cost too much memory if use 2 workers.